api/app/routes/admin.py

```
@@ -277,12 +307,227 @@ async def get_metrics():
     return Response(content=generate_latest(), media_type=CONTENT_TYPE_LATEST)
 
 
+@router.get("/feedback/list", response_model=FeedbackListResponse)
+async def get_feedback_list(
+    rating: str = None,
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)